tests.sh

```diff
@@ -1,78 +1,105 @@
 #!/bin/bash
 
-
-# Current test session
-TEST_DIR=""
-
-# Verbosity level
-TEST_VERBOSE=0
-
-# Assertions counter
-TEST_ASSERTS=0
-
-# Last stdout
-TEST_STDOUT=
-
-# Last stderr
-TEST_STDERR=
-
-# Current working directory for testcase.
-TEST_CASE_DIR=""
-
 # Public API Functions {{{
 
-# Function tests_tmpdir returns temporary directory for current test session.
-# Echo:
-#   Path to temp dir, e.g, /tmp/tests.XXXX
-tests_tmpdir() {
-    if [[ "$TEST_DIR" == "" ]]; then
-        tests_debug "test session not initialized"
+# @description Make all functions from tests.sh available without 'tests:'
+# prefix.
+#
+# @noargs
+tests:import-namespace() {
+    builtin eval $(declare -F | grep -F -- '-f tests:' | cut -d: -f2 |
+        sed -re's/.*/&() { tests:& "${@}"; };/'
+    )
+}
+
+# @description Returns temporary directory for current test session.
+#
+# It can be used as a workspace for the testcase.
+#
+# @example
+#   ls $(tests:get-tmp-dir)
+#
+# @echo Path to temp dir, e.g, /tmp/tests.XXXX
```

Comment:
hmm, may be `@output` instead of `@echo`? I think doc tags should be nouns, not verbs.